### ISSUE TITLE
_CREDS is not accessible from suites

### ISSUE THREAD

**Issue by lazyguru** [magento/magento2-functional-testing-framework#412]
When trying to access `_CREDS` from inside of a suite before section utilizing an action group that calls fillField you we receive the error:
```
FATAL ERROR. TESTS NOT FINISHED.
Uncaught Error: Class 'Group\CredentialStore' not found in vendor/magento/magento2-functional-testing-framework/src/Magento/FunctionalTestingFramework/Group/MyCustomSuite.php:70
```

The issue appears to be that the generated class does not import the full namespace for the CredentialStore class.  Manually adding that import to the generated class temporarily resolves the issue.

### Preconditions
1. Tested using Magento 2.3.2 with MFTF 2.3.14
2. Create a custom module that includes MFTF tests and a suite.xml file
3. Have a before block in the suite that calls an action group
4. The action group should either directly reference `_CREDS` or have arguments that are filled using `_CREDS`

### Steps to reproduce
1. Try to run the suite using `vendor/bin/mftf r:g MyCustomSuite`

### Expected result
1. Suite should executed and populate credentials allowing tests to run

### Actual result
1. See summary at top for error message.

**Comment by soumyau** [comment]
@lazyguru Quick update. The fix is done and ready for review by the team. We plan to deliver it as a part of MFTF 2.4.4

**Comment by lazyguru** [comment]
Will 2.4.4 be released with Magento 2.3.3?  Is there an upgrade guide for changes from 2.3.x to 2.4.x? (or are you following SemVer here and everything is backwards compatible?)

**Comment by soumyau** [comment]
Yes, MFTF follows SemVer and will be backward compatible with older versions. And yes, we are awaiting a go signal from PO to release it with Magneto 2.3.3. Will keep you updated.

**Comment by soumyau** [comment]
MFTF 2.4.4 is now available on 2.3.3-develop!

**Comment by KevinBKozan** [comment]
Closing since #415 was merged.
